src/graph.rs

```diff
@@ -546,11 +542,8 @@ impl<N, E, Ty, Ix> Graph<N, E, Ty, Ix>
     /// endpoint in the displaced node.
     pub fn remove_node(&mut self, a: NodeIndex<Ix>) -> Option<N>
     {
-        match self.nodes.get(a.index()) {
-            None => return None,
-            _ => {}
-        }
-        for d in DIRECTIONS.iter() {
+        if self.nodes.get(a.index()).is_none() { return None }
```

Comment:
I'd prefer if this was not a oneliner but a regular if with line break after { and ; after return None.